### DISCUSSION TITLE
Recommended options to train on a low-vram gpu ?

### DISCUSSION THREAD

**Discussion by razvanab** [gitmylo/audio-webui#77] [Q&A]
What are the recommended options to train on a NVIDIA GeForce GTX 1060 6 GB ?

**Comment by gitmylo** [comment]
If you're talking about RVC training, the only setting that will affect vram usage is batch size, you can lower it if you are running out of vram.

**Comment by razvanab** [comment]
I tried with 1 as and i still get error about not havening enough vram.
Will changing the Epochs size have any benefit?

**Reply by gitmylo** [reply]
If you're running out of vram, you might have some other models loaded still, the bomb button (💣) lets you unload models from vram, in case they're still loaded. Alternatively, you could restart the webui, and it will also unload the models, as it was shut down.

Epoch size is not an option you can change, you could reduce the amount of training data, but it would not have an effect on vram usage. Neither does the epoch count, if that's what you were referring to.

**Reply by razvanab** [reply]
I understand I will continue experimenting.
Thanks.
